### ISSUE TITLE
Špatný popisek perku Střelec

### ISSUE THREAD

**Issue by Milhousek** [prekladyher/fallout4-cestina#225]
![20171230215504_1](https://user-images.githubusercontent.com/34947824/34462421-e807ab80-ee43-11e7-9a4c-38156968cfd2.jpg)

Místo "Útok s neautomatickou zbraní" by asi mělo být "Útok s neautomatickou puškou". Jelikož pojem zbraň zahrnuje všechny typy střelných zbraní, přičemž tento perk se vztahuje k neautomatickým "dvouručným" zbraním - pušky, brokovnice, muškety apod.

**Comment by StoupaJ** [comment]
Já bych řekl, že se tenhle perk bude týkat pro příklad třeba i veteškvéru, kterej nepovažuju za pušku, ale spíš za verzi granátometu. Takže "zbraní" by bylo v tomhle případě příhodnější...

**Comment by pavelhoral** [comment]
Ano, ale originál mluví o *rifle* a zobecnění velmi pravděpodobně nebylo úmyslné. Navíc v LSCR (byť ty bych zrovna za etalon kvality nebral) se k tomuhle perku mluví o puškách.
